mathesar/api/exceptions/exception_converters.py

```diff
@@ -0,0 +1,21 @@
+from django.utils.encoding import force_str
+from rest_framework_friendly_errors.settings import FRIENDLY_EXCEPTION_DICT
+
+from mathesar.api.exceptions.error_codes import ErrorCodes
+
+
+def default_api_exception_converter(exc, response):
+    if isinstance(response.data, list):
+        converted_data = []
+        for data in response.data:
+            new_data = {'code': ErrorCodes.NonClassifiedError.value,
```

Comment:
please use consistent style, one key/value pair per line, none of the pairs should not be on the same line as curly braces. also applies to other instances.